<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpSubMenuWidget</class>
 <widget class="QWidget" name="HelpSubMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <widget class="QFrame" name="frame_content">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="aboutLabel">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>20</y>
      <width>400</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Segoe UI;
font-size: 24px;
color: #e7e7e7;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QLabel" name="versionLabel">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>60</y>
      <width>400</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Segoe UI;
font-size: 15px;
color: #e7e7e7;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="copyrightLabel">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>95</y>
      <width>400</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Segoe UI;
font-size: 15px;
color: #e7e7e7;
background-color:transparent;
text-align: right center;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
